stablehlo/transforms/VhloToVersion.cpp

```diff
@@ -261,11 +261,6 @@ struct VhloToVersionPass : public VhloToVersionPassBase<VhloToVersionPass> {
 /// Upgrade and Downgrade Infrastructure ///
 ////////////////////////////////////////////
 
-LogicalResult emitDowngradeError(Operation* op, llvm::StringRef message) {
-  return op->emitError("failed to downgrade ")
-         << op->getName() << ", " << message;
-}
-
 template <typename SourceOp, typename TargetOp>
 struct VersionConversionPattern : OpConversionPattern<SourceOp> {
```

Comment:
Hopefully no issues leaving this around and unused, maybe ok since its a template?

I'd rather not delete it since we'll likely need it in a week, so if compilation doesn't complain should be good.

Reply:
Looks like CI is happy, so let's keep it yeah.